src/crashreporter/crashui.cpp

```diff
@@ -0,0 +1,86 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "crashui.h"
```

Comment:
Typically, we try to organize the `#include` directives into three groups as follows:
1. Project-local includes (with the `.h` for this `.cpp` always coming first).
2. Qt includes
3. System global includes

And within each group the includes should be sorted alphabetically (unless there's a technical reason otherwise).